{"name": "Успешное создание курьера", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('courier_data', <FixtureRequest for <Function test_create_courier_success>>)", "trace": "file C:\\Users\\Пользователь\\sprint_7\\tests\\test_courier.py, line 12\n      @allure.title(\"Успешное создание курьера\")\n      @allure.severity(allure.severity_level.CRITICAL)\n      def test_create_courier_success(self, courier_data):\nE       fixture 'courier_data' not found\n>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, registered_courier, setup_courier_and_order, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nC:\\Users\\Пользователь\\sprint_7\\tests\\test_courier.py:12"}, "start": 1758527550117, "stop": 1758527550117, "uuid": "f60ceca5-e75d-4e4c-bfef-70c4ee8669ea", "historyId": "74949593bf2756bc0b392f5f4566cb6c", "testCaseId": "74949593bf2756bc0b392f5f4566cb6c", "fullName": "tests.test_courier.TestCourier#test_create_courier_success", "labels": [{"name": "epic", "value": "Courier Management"}, {"name": "feature", "value": "Courier API"}, {"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier"}, {"name": "subSuite", "value": "TestCourier"}, {"name": "host", "value": "HOME-PC"}, {"name": "thread", "value": "9396-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier"}]}